src/test/java/seedu/address/logic/NextCalendarNavigationCommandTest.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.logic;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.bidder.TypicalBidder.getTypicalBidderAddressBook;
+import static seedu.address.testutil.property.TypicalProperties.getTypicalPropertyBook;
+import static seedu.address.testutil.seller.TypicalSeller.getTypicalSellerAddressBook;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.calendarnavigation.NextCalendarNavigationCommand;
+import seedu.address.model.MeetingBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.bidbook.BidBook;
+
+// The test file is created after the email stating that
+// the packages should not be changed. Hence is it located in an existing
+// package.
+public class NextCalendarNavigationCommandTest {
+
+    private Model model;
+    private Model expectedModel;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(new UserPrefs(), new BidBook(),
+                getTypicalPropertyBook(), getTypicalBidderAddressBook(), getTypicalSellerAddressBook(),
+                new MeetingBook());
+        expectedModel = new ModelManager(new UserPrefs(), new BidBook(),
+                getTypicalPropertyBook(), getTypicalBidderAddressBook(), getTypicalSellerAddressBook(),
+                new MeetingBook());
+    }
+
+    // no changes should happen to any of the books
+    @Test
+    public void execute_listIsNotFiltered_showsSameList() {
+        assertCommandSuccess(new NextCalendarNavigationCommand(), model,
+                NextCalendarNavigationCommand.MESSAGE_SUCCESS, expectedModel);
+    }
+
```

Comment:
I dont understand what the test is for. Does navigation ever edit things in the models in the first place?

Reply:
Thanks, missed out the testing portion to include model inside